cypress/integration/spawn/host.ts

```diff
@@ -51,6 +51,23 @@ describe("Navigating to Spawn Host page", () => {
     cy.get('[data-row-key="i-092593689871a50dc"] > :nth-child(2)').click();
     cy.dataCy("spawn-host-card").should("not.be.visible");
   });
+  it("Hovering over copy ssh button should toggle tooltip", () => {
+    cy.get('[data-row-key="i-092593689871a50dc"] > :nth-child(7)').trigger(
+      "mouseover"
+    );
+    cy.dataCy("copy-ssh-tooltip").contains("Must be on VPN to connect to host");
+  });
+  it("Clicking on copy ssh button should change tooltip message", () => {
+    cy.get('[data-row-key="i-092593689871a50dc"] > :nth-child(7)').click();
+    cy.dataCy("copy-ssh-tooltip").contains("Copied!");
+  });
+  it("Waiting should change tooltip message back ", () => {
+    cy.wait(10000);
+    cy.get('[data-row-key="i-092593689871a50dc"] > :nth-child(7)').trigger(
+      "mouseover"
+    );
+    cy.dataCy("copy-ssh-tooltip").contains("Must be on VPN to connect to host");
+  });
   it("Visiting the spawn host page with an id in the url should open the page with the row expanded", () => {
```

Comment:
Hey i was actually referring to creating a unit test for the button, not a cypress test. Although this looks great its easier to test in react testing library and we don't actually want to add a 10 second delay to our tests. We can sync after stand up on how to turn these into unit tests

Reply:
unittest made